hs_core/views/__init__.py

```diff
@@ -1155,9 +1156,28 @@ def update(self, group_to_update, request):
 @processor_for('my-resources')
 @login_required
 def my_resources(request, page):
-
+    """
+    TODO parameterize and give user control of listings per page, saved in user preferences
+    Template processor for my-resources.html resources listing for user
+    :param request: WSGI GET request for endpoint
+    :param page: Mezzanine middleware in process_view sends <Page: My Resources>
+    :return: html template render with page of resources
+    """
     resource_collection = get_my_resources_list(request)
-    context = {'collection': resource_collection}
+    page = request.GET.get('page')
+    # TODO investigate iRODS performance issue; listing caps out at 10 resources per page, otherwise very slow loading
```

Comment:
I am not sure how it is related to iRODS. If there is any performance issue it has to be database query related.

Reply:
Understood. I conflated this issue with other iRODS performance issues I am facing when bulk creating/reading entries for CZO activities.